Mathlib/Computability/Partrec.lean

```diff
@@ -152,6 +152,7 @@ theorem rfindOpt_mono {α} {f : ℕ → Option α} (H : ∀ {a m n}, m ≤ n →
     simp at this; simp [this, get_mem]⟩
 #align nat.rfind_opt_mono Nat.rfindOpt_mono
 
+/-- Type of partially recursive partial functions `ℕ → ℕ` -/
```

Comment:
This seems wrong to me, this is not a `Type` at all but a `Prop`.

Reply:
:facepalm: Would this be better?
```suggestion
/-- Partially recursive partial functions `ℕ → ℕ` -/

Reply:
That's still not quite right, because `h : PartRec f` does not mean "`h` is a partially recursive function". Maybe
```suggestion
/-- `PartRec f` means that `f : ℕ → ℕ` is partially recursive. -/
```